src/Tests/Microsoft.DotNet.PackageValidation.Tests/CompatibleFrameworksInPackageTests.cs

```diff
@@ -86,7 +86,7 @@ public class First
             var result = packCommand.Execute();
             Assert.Equal(string.Empty, result.StdErr);
             Package package = Package.Create(packCommand.GetNuGetPackage(), null);
-            new CompatibleFrameworkInPackageValidator(_log).Validate(new() { Package = package });
+            new CompatibleFrameworkInPackageValidator(_log).Validate(new(package));
```

Comment:
NIT: I know you didn't add these here, but since you are already editing these lines, can we type explicitly the type for which we are invoking the constructor? We usually are fine with implicitly just using `new()` whenever you can infer the type from the left operand, like `Dictionary<int, int> d = new();` but in cases like this it makes the code harder to read as one would need to know what Validate() takes as a parameter.